webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -1378,9 +1378,9 @@ var CURRENT_GROUP_DATA={};
                 `+CUSTOM_CHAT_LAUNCHER+` 
                 </div>`
                   if (isAnonymousChat) {
-                      return '<a href="#" target="_self">' + CHAT_CLOSE_BUTTON + (CUSTOM_CHAT_LAUNCHER ? customLauncherHtml : defaultHtml) + '</a>';
+                      return '<a href="#" target="_self" aria-label="Open Kommunicate Chat" tabindex="0" role="button">' + CHAT_CLOSE_BUTTON + (CUSTOM_CHAT_LAUNCHER ? customLauncherHtml : defaultHtml) + '</a>';
```

Comment:
@CaptainT33mo can we remove "kommunicate" as for end users of the customer, it wouldn't be "Kommunicate chat"

Reply:
@devashishmamgain we can remove it but we need to add some title here otherwise the screenreaders will just read out the class names and HTML tag names.

Reply:
we can keep as Chat